bigchaindb/commands/utils.py

```diff
@@ -3,18 +3,39 @@
 """
 
 import argparse
+import builtins
+import functools
 import multiprocessing as mp
 import subprocess
+import sys
 
 import rethinkdb as r
 from pymongo import uri_parser
 
 import bigchaindb
+import bigchaindb.config_utils
 from bigchaindb import backend
 from bigchaindb.common.exceptions import StartupError
 from bigchaindb.version import __version__
 
 
+def configure_bigchaindb(command):
```

Comment:
Nice. Would be even cooler if instead of having a new function `configure_bigchaindb`, we make `autoconfigure` working both as a function and as a decorator (less cognitive overload).